src/autoscaled_pool.js

```diff
@@ -85,6 +88,11 @@ export default class AutoscaledPool {
         // which gets resolved once everything is done.
         this.resolve = null;
         this.reject = null;
+
+        // Connect to actor events for CPU info.
+        events.on(ACTOR_EVENT_NAMES.CPU_INFO, (data) => {
```

Comment:
on shutdown of pool, we should remove the listener

Reply:
y, right